### PR TITLE
Update issue-solution.md

### PR DESCRIPTION
**Pull request by it33**
#### Summary

Updating to new ISP template, training and process

### CODE REVIEW

File: operations/operations/issue-solution.md [hunk 1]
```diff
@@ -1,43 +1,10 @@
 # Issue/Solution Process
 
-Issue/Solution Proposal is a tool for increasing clarity, speed and output in teams. This page describes the template and process to be used.
+Issue/Solution Proposal is a tool for increasing clarity, speed and output in teams.
```
Comment (justinegeffen) [inline operations/operations/issue-solution.md]:
```suggestion
Issue/Solution Proposal is a tool for increasing clarity, speed, and output in teams.
```

File: operations/operations/issue-solution.md [hunk 2]
```diff
@@ -1,43 +1,10 @@
 # Issue/Solution Process
 
-Issue/Solution Proposal is a tool for increasing clarity, speed and output in teams. This page describes the template and process to be used.
+Issue/Solution Proposal is a tool for increasing clarity, speed and output in teams. 
 
-## Framework and process
+We are now on Version 3 of our process and the documentation, video training and ISP template are hosted internally here: https://docs.google.com/document/d/17oHNwYMg_iil3HpGgg1QBIqCX4Q9KoYAyqMvjfwTiGE/edit
```
Comment (justinegeffen) [inline operations/operations/issue-solution.md]:
```suggestion
We're now on version three of our process. The documentation, video training, and ISP template are hosted internally [here](https://docs.google.com/document/d/17oHNwYMg_iil3HpGgg1QBIqCX4Q9KoYAyqMvjfwTiGE/edit).
```

File: operations/operations/issue-solution.md [hunk 3]
```diff
@@ -1,43 +1,10 @@
...
-A listing of stakeholders and their written feedback on whether they support or do not support. If someone is not available, indicate that in their feedback section and seek feedback within 3 business days from the synchronous meeting or their return from [PTO](../../company/about-mattermost/list-of-terms.md#pto), whichever is later. If the stakeholder attended the meeting and did not decide to enter Feedback on Solution nor Feedback on Decision are blank, it is assumed the stakeholder did not have material objections to share.
-
-## Template
-
-The following templates can be used for Issue/Solution preparation by copying the template to a new file and completing the form.
-
-* [Generic Issue/Solution template](https://docs.google.com/document/d/1XDgumIgT525mH40dj24K3q_muIzJOVzHY5uVeg5OUkQ/edit) - General purpose Issue/Solution template.
-* [MLT Issue/Solution template](https://docs.google.com/document/d/1fNE-MzljPURKSYN4VrTaDz4T1JE3C0gnNCbHDpIlSRA/edit) - Template with Mattermost Leadership Team members pre-populated.
+Over time, this page in the handbook will be replaced with a [Focalboard](https://focalboard.com) page where a summary of our process and template will be hosted.
```
Comment (justinegeffen) [inline operations/operations/issue-solution.md]:
```suggestion
Over time, this page will be replaced with a [Focalboard](https://focalboard.com) page where a summary of our process and template will be hosted.
```

**Review by justinegeffen — COMMENTED** [review]
Reviewed content editorially and added some comments. Happy to commit them and approve as needed if this iteration is complete.

**Comment by it33** [comment]
LGTM

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**